_source/user-guide/infrastructure-monitoring/annotations.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: article
+title: Annotations
+permalink: /user-guide/infrastructure-monitoring/annotations/
+flags:
+  logzio-plan: community
+tags:
+  - metrics
+contributors:
+  - shalper
+  - daniel-tk
+---
+
+Annotations can mark events on your Metrics dashboards based on data from a logging account. For example, an annotation can indicate when a particular error log or configuration change occured based on logs written to your main logging account.
+
+Annotations offer another way to correlate your metrics with your logs, to help you figure out _why_ things are happening. Annotations are similar to using [Markers]({{site.baseurl}}/user-guide/insights/markers.html) in your logging [Insights]({{site.baseurl}}/user-guide/insights/exploring-insights.html).
+
+
+
+#### Configure an annotation
+
+<div class="tasklist">
+
+##### Pick your dashboard
+
+Before you begin, you'll need to make sure you have permissions to edit the dashboard.
+
+If you are using any of the pre-configured dashboards provided by Logz.io, you'll first need to select the option to make it editable. (You can also click the gear **<i class="li li-gear"></i>** in the top menu, then select **Save as > Save**).
+
+##### Add a new annotation
+
+Go to the dashboard's settings:
+
+* Click the gear **<i class="li li-gear"></i>** in the dashboard's top menu (upper right corner).
+* Select **Annotations** > **New**.
+* The annotation form will appear. Fill in the form, starting with the **General** section.
+  1. Give a short **Name**. The name should describe the significance of the event.
+  2. Select a logging account as the **Data source**. This is the relevant Kibana account.
```

Comment:
*Optional* Maybe we should also bold the word 'logging'? To stress out even more the fact that annotations work over logs and not metrics?

Reply:
@daniel-tk  We can, but I'm not sure it will help to clarify the point. Maybe I need to make the introduction stronger?